nat-lab/tests/utils/connection_tracker.py

```diff
@@ -178,96 +176,3 @@ async def run(self) -> AsyncIterator["ConnectionTracker"]:
                     await asyncio.sleep(0.1)
 
             yield self
-
-
-def generate_connection_tracker_config(
```

Comment:
Why does this need to move?

Reply:
because of circular dependency `connection_tracker <-> connection_util`